index.bs

```diff
@@ -470,6 +480,8 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
```

Comment:
Does `|inclusiveDescendants|` include the top-level document as well? https://html.spec.whatwg.org/multipage/browsers.html#list-of-the-descendant-browsing-contexts is not super clear on this

Reply:
Yes.
`|inclusiveDescendants|` includes the `|successorBC|`, because is was initialized by this line.

```
Let inclusiveDescendants be « successorBC » extended with successorBC’s active document's list of the descendant browsing contexts.
```

Reply:
Oh, ok. Would be better IMO to clarify this and reduce type confusion (out of scope for this PR), e.g. "Let inclusiveDescendants be an empty list. Append successorBC to inclusiveDescendants. Extend inclusiveDescendants with the list of descendant browsing contexts"